RevertConditions/ReturnSize/README.md

```diff
@@ -0,0 +1,28 @@
+# Revert On Return Size Example
+
+## Overview
+This repository contains a smart contract named `ReturnSize` that includes a function called `callerFunction`. This function takes a library address and a uint as arguments, relying on the implementation of `calledFunction` within the provided library address to execute certain logic based on the input uint. The purpose of this example is to demonstrate the utilization of the CVL prover argument flag `-superOptimisticReturnsize`. This flag ensures that the correct number of return values is obtained from a NONDET (non-deterministic) unresolved call, thereby preventing a revert scenario in such instances.
+
+**NOTE:** For additional information about the flag, please refer to the [Certora Documentation](https://docs.certora.com/en/latest/docs/prover/cli/options.html#prover-args-optimisticreturnsize-true).
```

Comment:
by adding the flag 'optmisticReturnSize' the Prover takes into account only values where return size is as expected